packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphRenderer.tsx

```diff
@@ -467,7 +467,11 @@ class FlameGraphRenderer extends React.Component<
               styles.panesWrapper
             )}`}
           >
-            {dataUnavailable ? <NoProfilingData /> : panes.map((pane) => pane)}
+            {dataUnavailable ? (
+              <NoData dataName="profiling" />
+            ) : (
+              panes.map((pane) => pane)
+            )}
```

Comment:
I think this can be simplified to simply `No data available`, without specifying a qualifier.

Reply:
`No data available` instead of `No profiling data available for this application / time range.` ?

Reply:
I think so, then we don't need all this new code. cc @Rperry2174

Reply:
there was a `NoProfilingData` component which is div + span + styles to add paddings and center text, i moved the whole component to the _webapp/ui_ folder....so its not that new code

Reply:
Ah I meant allowing custom title (`profile` | `heatmap`), that's a level of customization I don't think we really need it.